main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -254,27 +254,29 @@ object Packager {
     }
 
     // Construct a new zip file.
-    val zip = new ZipOutputStream(Files.newOutputStream(jarFile))
-
-    // META-INF/MANIFEST.MF
-    val manifest =
-      """Manifest-Version: 1.0
-        |Main-Class: Main
-        |""".stripMargin
-
-    // Add manifest file.
-    addToZip(zip, "META-INF/MANIFEST.MF", manifest.getBytes)
-
-    // Add all class files.
-    for (buildFile <- getAllFiles(getBuildDirectory(p))) {
-      val fileName = getBuildDirectory(p).relativize(buildFile).toString
-      val fileNameWithSlashes = fileName.replace('\\', '/')
-      addToZip(zip, fileNameWithSlashes, buildFile)
+    Using(new ZipOutputStream(Files.newOutputStream(jarFile))) { zip =>
+      // META-INF/MANIFEST.MF
+      val manifest =
+        """Manifest-Version: 1.0
+          |Main-Class: Main
+          |""".stripMargin
+
+      // Add manifest file.
+      addToZip(zip, "META-INF/MANIFEST.MF", manifest.getBytes)
+
+      // Add all class files.
+      for (buildFile <- getAllFiles(getBuildDirectory(p))) {
+        val fileName = getBuildDirectory(p).relativize(buildFile).toString
+        val fileNameWithSlashes = fileName.replace('\\', '/')
+        addToZip(zip, fileNameWithSlashes, buildFile)
+      }
+    } match {
+      case Success(()) => ().toOk
+      case Failure(e) => {
+        e.printStackTrace()
```

Comment:
@mlutze is it acceptable to use `Throwavle#printStackTrace()` to print the error message and stack trace? I think it's better than logger, because logger is not used in this Packager class.

I found other lines use `e => println(e.message(flix.getFormatter))`, but in this case `e` is not `Throwable` but `CompilationMessage`. In this PR we need to handle general throwables, so we cannot use this interface.

Reply:
One more question: which we should use to print error message: STDOUT or STDERR?
If we need to print error message via STDOUT, we should change this line as follows:

```suggestion
        e.printStackTrace(System.out)
```

Reply:
handled as d5954d3025ec4c45c2ba474e17df247158d33642, thanks!